### DISCUSSION TITLE
Burst-fire weapons

### DISCUSSION THREAD

**Discussion by SweptWasTaken** [space-wizards/space-station-14#4049] [General]
Would be nice.

Also ties into improvements to guns like #1867 

[I want my G11](https://youtu.be/E_didDgUjn0)

**Comment by metalgearsloth** [comment]
For reference for anyone looking at this it's already in but no guns actually use it as a default (main reason is no hotbar to change fire mode).

**Comment by SweptWasTaken** [comment]
Yes, i'll probably add a few myself, but they'll be stuck on burst-fire.
